source/speech/speech.py

```diff
@@ -1759,9 +1760,9 @@ def getControlFieldSpeech(  # noqa: C901
 			)
 		)
 	):
-		description = attrs.get('description', "")
+		description = attrs.get('description')
 	else:
-		description=""
+		description = None

```

Comment:
Note that we use `if description is not None` further down to generate the `descriptionSequence`